### DISCUSSION TITLE
Issues from starboard repository

### DISCUSSION THREAD

**Discussion by kaiquerass** [aquasecurity/trivy-operator#235] [Development]
Hi people! I'm not sure if this is the right place for this question, but the issues from starboard repository will be resolved here?

There is an open issue that still not working on trivy operator. Would be awesome to have only the truly active reports. 

https://github.com/aquasecurity/starboard/issues/926

**Comment by chen-keinan** [comment]
@kaiquerass I have moved the issue #237 to  ***trivy-operator*** repo  backlog , we will review it shortly.
